src/components/Header.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from "react";
+
+import '../styles/Header.scss';
+
+const logoDesktop = './images/logo-desktop.svg';
+const logoMobile = './images/logo-mobile.svg';
+const iconImportCSV = './images/importCSV.svg';
+const minDesktopWidth: number = 768;
+
+const Header = () => {
+    const [isResized, setIsResized] = useState(window.outerWidth >= minDesktopWidth);
+
+    const onResize = () => {
+        if (window.outerWidth >= minDesktopWidth) {
+            setIsResized(true);
+        } else {
+            setIsResized(false);
+        }
+    };
+
+    useEffect(() => {
```

Comment:
hm, are you going to use this hook only header?

Reply:
for now yes,  in a future possible we will separate this logic